dataflux_pytorch/benchmark/checkpointing/singlenode/train.py

```diff
@@ -103,6 +105,9 @@ def main():
     args = parse_args()
     if args.steps < 1:
         raise ValueError("Steps need to greater than 0.")
+    if args.no_lightning and args.no_dataflux_ckpt:
+        raise ValueError(
+            "Cannot specify --no-dataflux-ckpt and --no-lightning, pick one.")
```

Comment:
We will have a slight merge conflict with my args change in PR #135. I don't think it'll make sense to add `no-lightning` to the new checkpoint choice flag, so this block will likely be changed to something like:

```python
if args.checkpoint == "no-dataflux" and args.no_lightning:
    ...
```
    

https://github.com/GoogleCloudPlatform/dataflux-pytorch/pull/135/files#diff-daaa477a49e483154fd42d8638792f354f98cd26b79f5c5c4542582757b56fbbR85-R88

Reply:
yeah i saw this on your PR, will update